src/platform/plugins/shared/unified_doc_viewer/public/index.tsx

```diff
@@ -39,5 +39,5 @@ export { UnifiedDocViewerLogsOverview } from './components/lazy_doc_viewer_logs_
 export { UnifiedDocViewerObservabilityTracesSpanOverview } from './components/observability/traces/doc_viewer_span_overview/lazy_doc_viewer_obs_traces_span_overview';
 export { UnifiedDocViewerObservabilityTracesTransactionOverview } from './components/observability/traces/doc_viewer_transaction_overview/lazy_doc_viewer_obs_traces_transaction_overview';
 export { UnifiedDocViewerObservabilityAttributesOverview } from './components/observability/attributes/doc_viewer_attributes_overview/lazy_doc_viewer_obs_attributes_overview';
-
+export type { TraceIndexes } from './components/observability/traces/hooks/use_data_sources';
```

Comment:
Nit: Might make sense to export from somewhere like `kbn-discover-utils` if it'll be used in multiple places, not really sure the intent for it though.